src/internal/components/ThemeEditor.tsx

```diff
@@ -108,6 +110,7 @@ export const ThemeEditor = (props: ThemeEditorProps) => {
         setThemeInitialHistory({
           history: [themeData],
           index: 0,
+          hash: uuidv4(),
```

Comment:
It's ok here, but also needs to be part of the history themeData object. See how we do it for initialPuckHistory (which this whole func basically emulates). 

```
setPuckInitialHistory({
          histories: [{ id: "root", state: { data: visualConfigurationData } }],
          appendData: false,
        });
```
id === hash here. We could just call it "root" as well.